metrics/network/remote_network/remote-networking-iperf3.sh

```diff
@@ -128,9 +151,12 @@ function main {
 		remote_network_bandwidth_iperf3
 	elif [ "$jitter_test" == "1" ]; then
 		remote_network_jitter_iperf3
+	elif [ "$parallel_test" == "1" ]; then
+		remote_network_parallel_iperf3
```

Comment:
why do you verify a var for calling a function and it is not called inside the case?